packages/ui/src/helpers/accountSettings/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { AmplifyUser } from '../../types';
+import { Auth } from 'aws-amplify';
+
+type ChangePasswordInput = {
+  user: AmplifyUser;
+  currentPassword: string;
+  newPassword: string;
+};
+
+export const changePassword = async ({
+  user,
+  currentPassword,
+  newPassword,
+}: ChangePasswordInput): Promise<void> => {
+  try {
+    /**
+     * Auth.changePassword returns `Promise<"SUCCESS">`. We're not interested
+     * in its resolved string value, so we just return Promise.resolve() on success.
+     */
+    await Auth.changePassword(user, currentPassword, newPassword);
```

Comment:
Not super important/non-blocking,  but what do you think of improving our logging for utilities that call Amplify JS?

Reply:
Added logger util in [6a9d385](https://github.com/aws-amplify/amplify-ui/pull/2885/commits/6a9d3850a7f8d85e67aed74ed876073934f16468)